static/styles/zulip.css

```diff
@@ -1440,6 +1440,7 @@ div.focused_table {
     margin: 0 5px 5px 0 !important;
     align-self: center;
     width: 100%;
+    white-space: nowrap;
```

Comment:
Can you post screenshots showing what these changes do visually?

Reply:
before:
<img width="499" alt="Screenshot 2020-10-03 at 9 01 49 AM" src="https://user-images.githubusercontent.com/25124304/94982317-1fe00200-0557-11eb-80f1-0616111e1e3b.png">

after:
<img width="499" alt="Screenshot 2020-10-03 at 9 01 57 AM" src="https://user-images.githubusercontent.com/25124304/94982318-25d5e300-0557-11eb-8a6e-13073b795432.png">